genon/preprocessor/facade/attachment_processor_guardrail.py

```
@@ -0,0 +1,1626 @@
+def guardrail(func):
+    @wraps(func)
+    async def wrapper(*args, **kwargs):
+        result = await func(*args, **kwargs)
+        for r in result:
+            url = f"{GENOS_URL}/workflow/{GUARDRAIL_WORKFLOW_ID}"
+            headers = dict(Authorization=f"Bearer {GUARDRAIL_BEARER_TOKEN}")
+
+            if hasattr(r, "text"):
+                body = {'question': r.text}
+
+                res = requests.post(f'{url}/run/v2', json=body, headers=headers)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)